aptos-move/e2e-move-tests/tests/code_publishing.rs

```diff
@@ -18,7 +18,7 @@ struct State {
 
 #[test]
 fn code_publishing_basic() {
-    let mut h = MoveHarness::new();
+    let mut h = MoveHarness::new_no_parallel();
```

Comment:
Why no parallel?

Reply:
Because code publishing doesn't work with it yet. I added a comment which comes with the next committ.